src/FirstSteps/RPG/Items/OldChest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FirstSteps.RPG.Items
+{
+    class OldChest
+    {
+        private static List<Item> itemsForHuman = new List<Item>()
+        {
+            new Pitchfork(),
+            new Knife()
+        };
+        public static Item Open()
+        {
+            var random = new Random().Next(0,5);
+            if(random > itemsForHuman.Count())
+            {
+                Console.WriteLine("\nUnfortunately, Empty slot!");
```

Comment:
In reference to this comment `Maybe at first try to display the name of the item instead of adding it to the backpack.`. Move this line to the `Village` class.